src/Audit/Audit.php

```diff
@@ -2,19 +2,47 @@
 
 namespace Utopia\Audit;
 
+use Utopia\Database\Database;
+use Utopia\Database\Document;
+use Utopia\Database\Query;
+use Utopia\Database\Validator\Authorization;
+
 class Audit
 {
+    const COLLECTION = "abuse";
     /**
-     * @var Adapter
+     * @var Database
      */
-    private $adapter;
+    private $db;
 
     /**
-     * @param Adapter $adapter
+     * @param Database $adapter
      */
-    public function __construct(Adapter $adapter)
+    public function __construct(Database $db)
+    {
+        $this->db = $db;
+        $this->init();
+    }
+
+    private function init(): void
```

Comment:
Double thinking about it, I think `install` or `setup` might better describe this method.